src/Tizen.NUI.Components/Style/Navigation/AppBarStyle.cs

```diff
@@ -41,36 +41,36 @@ public AppBarStyle() : base()
         /// Creates a new instance of a AppBarStyle with style.
```

Comment:
style with style? could you tell more? maybe new style is copy, inherits settings?

Reply:
Thank you for your comment.
The given style is copied to the new instance.
The description has been updated.